Scalar/CommandLine/ScalarVerb.cs

```diff
@@ -492,9 +492,14 @@ private string GetAlternatesPath(ScalarEnlistment enlistment)
         private void CheckGitVersion(ITracer tracer, ScalarEnlistment enlistment, out string version)
         {
             GitVersion gitVersion = null;
-            if (string.IsNullOrEmpty(enlistment.GitBinPath) || !GitProcess.TryGetVersion(enlistment.GitBinPath, out gitVersion, out string _))
+            if (string.IsNullOrEmpty(enlistment.GitBinPath))
             {
-                this.ReportErrorAndExit(tracer, "Error: Unable to retrieve the git version");
+                this.ReportErrorAndExit(tracer, "Unable to find Git version on PATH");
+            }
+
+            if (!GitProcess.TryGetVersion(enlistment.GitBinPath, out gitVersion, out string error))
+            {
+                this.ReportErrorAndExit(tracer, $"Unable to parse Git version at '{enlistment.GitBinPath}':\n{error}");
             }
 
             version = gitVersion.ToString();
```

Comment:
Would it help to also print in the next 2 error messages the pathname to the git.exe that we inoked?